pkg/tsdb/azuremonitor/routes.go

```diff
@@ -49,6 +53,18 @@ var azUSGovLogAnalytics = types.AzRoute{
 	Headers: map[string]string{"x-ms-app": "Grafana", "Cache-Control": "public, max-age=60"},
 }
 
+var azPortal = types.AzRoute{
+	URL: "https://management.azure.com",
```

Comment:
I believe all three of these portal URLs are incorrect. Should they not be:
```
"https://portal.azure.com"
"https://portal.azure.cn"
"https://portal.azure.us"
```

Reply:
You are right, I overlooked it.